test/example-types/metricGroups/pivot.test.js

```diff
@@ -827,10 +817,6 @@ describe('pivot', () => {
                         from: '0',
                         to: '500',
                       },
-                      {
-                        from: '500',
-                        to: '10000',
-                      },
```

Comment:
this seems like a regression - don't we need both buckets?

Reply:
The number range expansions were also broken because they were returning more buckets than the pivot implementation was expecting.  Per expansion request there should only be a single bucket each time.

Reply:
This should be a fix as far as I know since it was an outstanding issue.

Reply:
@daedalus28  This one is not a regression it's a fix. When we drill down we shouldn't aggregate ranges that are not within the drilldown. It returns empty arrays as a result and breaks UI. This PR is fixing exactly that